src/srcShared/fs.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+
+const fs = vscode.workspace.fs
+type Uri = vscode.Uri
+
+/**
+ * @warning Do not import this class specifically, instead import the instance {@link fsCommon}.
```

Comment:
could use a private constructore, and this instance pattern that is used in a few places already: https://github.com/aws/aws-toolkit-vscode/blob/4dcee33931693380739eaa5d44e92fa4545a9666/src/shared/settings.ts#L167-L174

Reply:
I did the export of the instance so that for each use we don't always need to call `FileSystem.instance.{action}()` and could instead do `fsCommon.{action}()`, similar to `fs`. But makes sense to follow what we've already been doing, will update.